mitmproxy/addons/multipart.py

```diff
@@ -0,0 +1,43 @@
+from urllib.request import Request, urlopen
+from urllib.error import HTTPError, URLError
+import mimetypes
+
+
+def get_content_type(filename):
+    return mimetypes.guess_type(filename)[0] or 'application/octet-stream'
```

Comment:
This isn't really necessary, the content type is constant.

Reply:
Yeah okay